scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js

```diff
@@ -0,0 +1,82 @@
+const { spawn } = require('child_process');
+
+const API_URL = 'https://sheetdb.io/api/v1/g3uyrf24ruo1v';
```

Comment:
The `g3uyrf24ruo1v` part of the URL looks like a secret/token. If so, it should be invalidated, and new token should be passed via [GitHub encrypted secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) mechanism.

It's a public repo, so no secrets should be committed to it.

Reply:
Actually, why do we need to use some external sheets if we can save the output in the current repo?